doc/source/ray-core/objects/limiting-memory-usage.rst

```diff
@@ -0,0 +1,274 @@
+Limiting the memory usage
+=========================
+
+The application may consume a large amount of heap space and causes the node to run out of memory. When that happens the operating system will start killing worker or raylet processes, which breaks Ray. It may stall the metrics and if this happens on the head node, it may stall the :ref:`dashboard <ray-dashboard>` or other control processes and causes Ray to fail.
+
+Memory Monitor
+--------------
+
+The memory monitor is a component that runs on each node. It periodically checks the memory usage, which includes the worker heap, the object store, and the raylet as described in :ref:`memory management <memory>`. If the combined usage exceeds a configurable threshold the raylet will kill task or actor to free up memory and prevent Ray from failing.
+
+.. note::
+
+    The memory monitor is in :ref:`alpha <api-stability-alpha>`. It is disabled by default and needs to be enabled by setting the environment variable ``RAY_memory_monitor_interval_ms`` to a value greater than zero when Ray starts. It is available on Linux and is tested with Ray running inside a container that is using cgroup v1. If you encounter issues when running the memory monitor outside of a container or the container is using cgroup v2, please :ref:`file an issue or post a question <limiting-memory-usage-questions>`. 
+
+Enabling the Memory Monitor
+---------------------------
+
+Verify the memory monitor is disabled by default:
```

Comment:
Not sure if we need instructions for this since it's already the default? It's probably sufficient to just give the below instructions for enabling, and then give the alternative output if it's disabled.